frontend/src/store/auth.js

```diff
@@ -52,12 +55,53 @@ export const useAuthStore = defineStore('auth', {
     removeFromLocalStorage() {
       localStorage.removeItem('token');
       localStorage.removeItem('userAuth');
+      localStorage.removeItem('signInTries');
+      localStorage.removeItem('signInSecondsAcumulated');
+      localStorage.removeItem('signInSecondsRemaining');
     },
     /**
      * Clears the authorization header from Axios.
      */
     clearAuthorizationHeader() {
       delete axios.defaults.headers.common['Authorization'];
+    },
+     /**
+     * Saves the Sign In Attemps and Second Remaining to localStorage
+     */
+    saveToLocalStorageSignIn() {
+      localStorage.setItem('signInTries', this.signInTries);
+      localStorage.setItem('signInSecondsRemaining', this.signInSecondsRemaining);
+      localStorage.setItem('signInSecondsAcumulated', this.signInSecondsAcumulated);
+    },
+     /**
+     * Verficate the attemps to Sign In and incrementate the seconds for wait the next 3 attemps
+     */
+    attempsSignIn(action) {
+      if (action != 'initial'){
+        this.signInTries += 1;
+      }
+      console.log(this.signInTries)
```

Comment:
You should remove all **console.log()** statements, just use them to test purposes